src/main/resources/logback.xml

```diff
@@ -0,0 +1,26 @@
+<configuration>
+    <import class="ch.qos.logback.classic.encoder.PatternLayoutEncoder" />
+    <import class="ch.qos.logback.core.ConsoleAppender" />
+
+    <property name="LOG_PATH" value="target/log"/>
+    <property name="PATTERN" value="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
```

Comment:
Add the method title for log prefix as well.

Reply:
Fix